src/components/usa-card/usa-card~standard.json

```diff
@@ -0,0 +1,61 @@
+{
+  "modifier": "",
+  "default_classes": "tablet:grid-col",
```

Comment:
Looks like 'grid-col' should be 'grid-col-4'
Reference: https://designsystem.digital.gov/components/card/